app/presenters/activity_csv_presenter.rb

```diff
@@ -12,4 +12,10 @@ def country_delivery_partners
     return if super.blank?
     super.join(" | ")
   end
+
+  def implementing_organisations
+    return if super.empty?
+    implementing_organisations = super.collect { |implementing_organisation| implementing_organisation.name }
```

Comment:
You can probably achieve the same with a one-liner:

```suggestion
    super.pluck(:name).join("|")
```

Reply:
Gah! I knew there would be a one-liner, I just couldn't get to it! Thank you! I will use this 👍